lldb/test/Shell/Recognizer/SwiftRuntimeFailure.test

```diff
@@ -0,0 +1,10 @@
+# UNSUPPORTED: system-windows
+# RUN: %target-swift-frontend -g %S/Inputs/RuntimeFailure.swift -Xllvm -enable-trap-debug-info -o %t.out
+# RUN: %lldb -b -s %s %t.out | FileCheck %s
+run
+# CHECK: thread #{{.*}}stop reason = Swift runtime failure: arithmetic overflow
+frame info
+# CHECK: frame #1{{.*}}`testit{{.*}} at RuntimeFailure.swift
+frame recognizer info 0
+# CHECK: frame 0 is recognized by Swift Runtime Failure StackFrame Recognizer
+q
```

Comment:
I think this should really be a python test, so it's tested with dwarf+dsym and on other platforms

Reply:
You can start by copying `test/lang/swift/any/`